src/test/scala/com/danielasfregola/twitter4s/helpers/JsonDiffSupport.scala

```diff
@@ -0,0 +1,87 @@
+package com.danielasfregola.twitter4s.helpers
+
+import java.util.Date
+
+import com.danielasfregola.twitter4s.http.serializers.JsonSupport
+import org.json4s.JsonAST._
+import org.json4s.{Diff, JField, JNothing, JObject}
+
+/**
+ * This is a copy of [[org.json4s.Diff]], with modifications to make it
+ * treat equivalent dates in different timezones as equal, and empty arrays
+ * as equal to nulls.
+ */
+trait JsonDiffSupport { this: JsonSupport =>
```

Comment:
OMG this is awesome!!!

Reply:
Are there any licence issues with copying this over from `org.json4s.Diff`, or is crediting it in the class comment sufficient?

Reply:
Hi @catalin-ursachi, they use Apache 2.0 like us...so give them credit should be enough!